_playwright-tests/tests/API/PopularRepos.spec.ts

```diff
@@ -54,4 +61,41 @@ test.describe('Popular repositories', () => {
       expect(beyondData.meta?.count).toBe(3); // Total count should still be 3
     });
   });
+
+  test('Test that searching rpms in non-added popular repo does not return an empty list', async ({
+    client,
+  }) => {
+    const epelUrl = 'https://dl.fedoraproject.org/pub/epel/10/Everything/x86_64/';
+    let rpmSearch: ApiSearchRpmResponse[];
+
+    await test.step('Check if repository already exists and delete it', async () => {
+      const existingRepos = await new RepositoriesApi(client).listRepositories(<
```

Comment:
Not able to list the epel repo :(

Reply:
The error I see in the CI is that the epel-release package isn't found. To fix that, @xbhouse is right, the repository is not introspected. You have to add this command to the actions as well: 

```
name: Introspect EPEL10 repo
run: go run cmd/external-repos/main.go introspect --url https://dl.fedoraproject.org/pub/epel/10/Everything/x86_64/ --force
````

Reply:
Done